File: src/main/java/csorba/norberth/lab1/models/Americano.java
package csorba.norberth.lab1.models;

import java.util.List;

public class Americano extends Coffee {
    public Americano(List<Ingredient> ingredients) {
        super(ingredients);
    }

    @Override
    public String toString() {
        return super.toString() + " > Americano";
    }
}

File: Laboratoare_gbir2109/DP_Lab1_Homework/DP_Lab1_Homework/Models/Americano.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DP_Lab1_Homework.Models
{
    public class Americano : BaseCoffee
    {
        public Americano()
        {
            Ingredients = new List<Ingredient>();
            CoffeeType = Coffee.Americano;
        }
    }
}
